swap/cashout.go

```diff
@@ -34,18 +35,21 @@ const CashChequeBeneficiaryTransactionCost = 50000
 type CashoutProcessor struct {
 	backend    chain.Backend     // ethereum backend to use
 	privateKey *ecdsa.PrivateKey // private key to use
+	Logger     Logger
 }
 
 // CashoutRequest represents a request for a cashout operation
 type CashoutRequest struct {
 	Cheque      Cheque         // cheque to be cashed
 	Destination common.Address // destination for the payout
+	Logger      Logger
```

Comment:
FYI `CashoutRequest ` and `ActiveCashout` were intended to be serializable structures (hence the upper case). There shouldn't really be a logger on there. Anyway this will change soon anyway, so you can leave it this way.